cfgov/cfgov/urls.py

```diff
@@ -427,9 +427,9 @@
 
     url(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
-    flagged_url('FWB_RELEASE',
-                r'^(?i)consumer-tools/financial-well-being/',
-                include('wellbeing.urls')
+    url(
+        r'^(?i)consumer-tools/financial-well-being/',
+        include_if_app_enabled('wellbeing.urls')
```

Comment:
@jimmynotjim If we leave it as `include_if_app_enabled`, it needs a `'wellbeing'` argument preceding `'wellbeing.urls'`.

Reply:
Whoops, thanks for catching that